DFC.ServiceTaxonomy.ContentApproval/DFC.ServiceTaxonomy.ContentApproval.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Razor">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <AddRazorSupportForMvc>true</AddRazorSupportForMvc>
+  </PropertyGroup>
```

Comment:
As it's a new project, would you mind adding the following...
```
    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
    <Nullable>enable</Nullable>
    <EnableNETAnalyzers>true</EnableNETAnalyzers>
    <AnalysisLevel>latest</AnalysisLevel>
```
and also these static analysis packages...
```
AsyncFixer
SonarAnalyzer.CSharp
```